mypy/newsemanal/semanal_namedtuple.py

```diff
@@ -189,15 +189,17 @@ def check_namedtuple(self,
         else:
             default_items = {}
         info = self.build_namedtuple_typeinfo(name, items, types, default_items)
-        # Store it as a global just in case it would remain anonymous.
-        # (Or in the nearest class if there is one.)
-        self.store_namedtuple_info(info, var_name or name, call, is_typed)
+        if var_name:
+            self.store_namedtuple_info(info, var_name, call, is_typed)
+        if name != var_name:
+            # Store function level (or broken) named tuple also in a name
```

Comment:
Can this also happen if we have an inline named tuple definition in a base class?

Also, can you explain in more detail how the symbol tables look like in this case (there are two symbol table entries, potentially with different names)?

Reply:
> Can this also happen if we have an inline named tuple definition in a base class?

Oh yes, forgot to mention this (we have a test case however). I will a detailed comment with some examples.